README.md

```diff
@@ -81,6 +81,13 @@ You can build standalone apps that package the server to run locally against SQL
     * `make win-wpf-app`
     * run `cd win-wpf/msix && focalboard.exe`
     * *Requires: Windows 10*
+* Docker:
+    * To run it localy from Offical Image
+    * `docker run -it -p 80:8000 mattermost/focalboard`
+    * To Build it for your Current Architekture
```

Comment:
Should `Architekture` not be  `architecture` ?

Reply:
Well it should have been :)
Sorry thats German, i glossed over that.

i hope its okay if i dont do a new Pull Request just for that Typo.
Or is there another way to Fix that easily (sorry not very deep Into Gihub yet) :)

Reply:
Well, I'm not working on this project at all 😅
I just passed by here because of an issue, and I noticed your typo, so I shared it with you 😁

But yeah sadly I don't think there is any other solution than creating a new PR to fix this...
Out of someone with write access on master to edit it right away 😹